admin_manual/configuration_server/occ_command.rst

```diff
@@ -552,6 +551,8 @@ This example will list all calendars for user annie::
 
  sudo -u www-data php occ dav:list-calendars annie
 
+``dav:dav:fix-missing-caldav-changes [user]`` Can try to restore calendar sync changes when data in the calendarchanges table has been lost. If the user ID is omitted, the command runs for all users. This can take a while.
```

Comment:
```suggestion
``dav:dav:fix-missing-caldav-changes [user]`` tries to restore calendar sync changes when data in the calendarchanges table has been lost. If the user ID is omitted, the command runs for all users. This can take a while.
```